src/ert/ensemble_evaluator/_builder/_realization.py

```diff
@@ -55,17 +55,43 @@ def set_iens(self, iens: int) -> Self:
         self._iens = iens
         return self
 
+    def set_max_runtime(self, max_runtime: Optional[int]) -> Self:
```

Comment:
I think you are missing `max_runtime: Optional[int] = None` if you want to keep `Optional` there

Reply:
Hmm, don't think that is needed, that would just allow us to do `builder.set_max_runtime()` which does not really make sense. But we might want to do `builder.set_max_runtime(None)`.

Reply:
Right, so being explicit is fine by me :+1: